webpack.prod.js

```diff
@@ -42,12 +42,25 @@ module.exports = merge(commons, {
     new webpack.BannerPlugin(`${process.env.npm_package_name} v${process.env.npm_package_version}`)
   ],
   optimization: {
+    runtimeChunk: 'single',
+    splitChunks: {
+      chunks: 'all',
+      maxInitialRequests: Infinity,
+      minSize: 200000,
+      cacheGroups: {
+        vendor: {
+          test: /[\\/]node_modules[\\/]/,
+          name(module) {
+            const packageName = module.context.match(/[\\/]node_modules[\\/](.*?)([\\/]|$)/)[1];
+
+            return `${packageName.replace('@', '')}`;
```

Comment:
Why is this needed?

Reply:
Same server can't serve files with an @ in the name